test/runnable/future.d

```diff
@@ -0,0 +1,43 @@
+/* PERMUTE_ARGS:
+ */
+
+class A
+{
+    @future char msg() { return 'a'; }
```

Comment:
If an attribute is required. What about making it a compiler recognized UDA. Declare it in `core.attributes`, mark it `package` so it's only available within druntime?